src/__tests__/print.ts

```diff
@@ -0,0 +1,86 @@
+import { createConsole, getLog, mockConsole } from 'console-testing-library';
+import { Context } from '../index';
+import { printResults } from '../print';
+
+describe('printResults', () => {
+  const expectedContext = {
+    version: '1.0.0',
+    cwd: 'cwd/string',
+    entry: ['src/client/main.js'],
+  } as Context;
+
+  let restore: any;
+  beforeEach(() => {
+    const strippingConsole = createConsole({ stripAnsi: true });
+    restore = mockConsole(strippingConsole);
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+    restore();
+  });
+
+  it('should print once to console if results are clean', () => {
+    const expectedProcessedResult = {
+      unresolved: [],
+      unimported: [],
+      unused: [],
+      clean: true,
+    };
+    printResults(expectedProcessedResult, expectedContext);
+
+    expect(getLog().log).toMatchInlineSnapshot(
+      `"✓ There don't seem to be any unimported files."`,
+    );
+  });
+
+  it('should print summary and unresolved, unimported, and unused tables populated', () => {
+    const expectedProcessedResult = {
+      unresolved: ['string', 'string'],
+      unimported: ['string', 'string', 'string', 'string'],
+      unused: ['string', 'string', 'string'],
+      clean: false,
+    };
+    printResults(expectedProcessedResult, expectedContext);
+
+    expect(getLog().log).toMatchInlineSnapshot(`
+      "
+             summary               unimported v1.0.0
```

Comment:
This test will fail after each version bump. Should we read the version from `package.json`, and inline that here?

```js
const version = require('package.json').version

    expect(getLog().log).toMatchInlineSnapshot(`
      "
             summary               unimported v${version}
```